main.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/skuid/keyman/server"
+	"github.com/skuid/keyman/sign"
+	"github.com/skuid/spec"
+	"github.com/skuid/spec/lifecycle"
+	_ "github.com/skuid/spec/metrics"
+	"github.com/skuid/spec/middlewares"
+	flag "github.com/spf13/pflag"
+	"github.com/spf13/viper"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+func main() {
+	// flags
+	level := spec.LevelPflagPCommandLine("level", "l", zapcore.InfoLevel, "Log level")
+	flag.StringP("key", "k", "./ca", "The CA private key to load")
+	flag.IntP("port", "p", 3000, "The port to listen on")
+
+	flag.Parse()
+
+	// Logging
+	l, err := spec.NewStandardLevelLogger(*level)
+	if err != nil {
+		zap.L().Fatal("Error initializing logger", zap.Error(err))
+	}
+	zap.ReplaceGlobals(l)
+
+	// Viper binds
+	viper.BindPFlags(flag.CommandLine)
+	viper.SetEnvPrefix("keyman")
+	viper.AutomaticEnv()
+	viper.SetEnvKeyReplacer(strings.NewReplacer("-", "_"))
```

Comment:
We should consider making this a convenience method in out spec library.

Reply:
yea I'm not opposed to that